bundler.go

```diff
@@ -41,6 +41,8 @@ type Configuration struct {
 	IconPathLinux   string `json:"icon_path_linux"`
 	IconPathWindows string `json:"icon_path_windows"` // .ico
 
+	HideLSUIElement bool `json:"hide_lsui_element"` // osx/macOS LSUIElement
```

Comment:
1) Could you rename this attribute to `DarwinAgentApp` since this is the desired purpose. Don't forget to update the json tag to `darwin_agent_app` and remove the comment on the right. I'd rather not have the `LSUIElement` technical name in the Configuration struct.
2) Could you order attributes by alphabetical order